pyccl/pyutils.py

```diff
@@ -664,3 +664,19 @@ def _get_spline3d_arrays(gsl_spline, length):
     check(status)
 
     return xarr, yarr, zarr.reshape((length, x_size, y_size))
+
+
+def check_openmp_threads():
+    openmp_threads = lib.openmp_threads()
+    if openmp_threads > 0:
+        print(f'OpenMP is working with {openmp_threads} threads max')
+    else:
+        print('OpenMP is not supported')
+
+
+def check_openmp_version():
+    openmp_version = lib.openmp_version()
+    if openmp_version > 0:
+        print(f'OpenMP version: {openmp_version}')
+    else:
+        print('OpenMP is not supported')
```

Comment:
I think it would be preferable to have functions that return the numerical values with the expected values documented in a doc string, rather than print messages to stdout.
Right now they can't be used in automated scripts, for example.